CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 5.0.0    2023-12-22
+
+* Align versioning scheme with that of other pg_query libraries
+  (which is to generally aim to match the libpg_query version)
+* Upgrade to libpg_query 5.0.0
+  - Updates to the Postgres 16 parser
+  - Multiple deparser improvements
+
```

Comment:
Shouldn't we include the `_opt` functions added for alternative parse modes since that's already available upstream?

Reply:
Hmm, I feel like we could maybe do that once we add support for calling them? (since right now we still use the "basic" versions of the functions)